### DISCUSSION TITLE
Using VSCode's command palette command

### DISCUSSION THREAD

**Discussion by y-arjun-y** [community/community#24060] [Other Feature Feedback, Questions, & Ideas]
<p>For consistency (as GitHub and VSCode are both by Microsoft and many programmers who are on GitHub use VSCode), why not have the command Cmd/Ctrl+Shift+P? (not Cmd/Ctrl+P since that is print on most browsers and the shift variation doesn’t seem to do anything on Chrome)</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks for sharing your feedback <a class="mention" href="/y-arjun-y">@y-arjun-y</a>! The keyboard shortcut has been quite the hot topic over the last day and we’ll be discussing this feedback as a team</p>

**Comment by bradzacher** [comment]
<p>Definitely agree. Having to remember both combos will be a pain point for users and also make it hard for people to discover it.</p>
<p>For reference - both Chrome devtools and Sublime also use cmd + shift + P for their command palette, so you’d be aligning with an existing standard across multiple common dev tools.</p>

**Comment by stefandesu** [comment]
<p>Came here to suggest that it would be nice to change the shortcut, particularly to Command + Shift + P. Maybe you could allow people to set it themselves?</p>

**Comment by y-arjun-y** [comment]
<p>Yes, having a way to change it would also be nice.</p>
